src/collection/algorithms/biconnected.js

```diff
@@ -0,0 +1,85 @@
+let elesfn = ({
+  biconnected: function() {
+
+    let eles = this;
+    let nodes = {};
+    let id = 0;
+    let edgeCount = 0;
+    let blocks = [];
+    let stack = [];
+    let visitedEdges = {};
+
+    const buildBlock = (x, y) => {
+      let i = stack.length-1;
+      let block = [];
+      while (stack[i].x != x || stack[i].y != y) {
+        block.push(stack.pop().edge);
+        i--;
+      }
+      block.push(stack.pop().edge);
+      blocks.push(this.spawn(block));
+    };
+
+    const biconnectedSearch = (root, currentNode, parent) => {
+      if (root == parent) edgeCount += 1;
+      nodes[currentNode] = {
+        id : id,
+        low : id++,
+        cutVertex : false
+      };
+      let sourceId, targetId, otherNodeId, edgeId;
+      let edges = eles.getElementById(currentNode).connectedEdges();
```

Comment:
In order to consider only elements in the calling collection, `connectedEdges()` must be filtered by the intersection with `eles`.

Reply:
Good catch.